cmake/libwebrtcbuild.cmake

```diff
@@ -58,6 +58,13 @@ elseif (APPLE)
         WEBRTC_POSIX
         WEBRTC_MAC
     )
+elseif (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
```

Comment:
maybe
```
if (WIN32)
    target_compile_definitions(libwebrtcbuild
    INTERFACE
        WEBRTC_WIN
    )
else()
    target_compile_definitions(libwebrtcbuild
    INTERFACE
        WEBRTC_POSIX
    )

    if (APPLE)
        target_compile_definitions(libwebrtcbuild
        INTERFACE
            WEBRTC_MAC
        )
    else()
        target_compile_definitions(libwebrtcbuild
        INTERFACE
            WEBRTC_USE_X11
        )
    endif()

    if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
        target_compile_definitions(libwebrtcbuild
        INTERFACE
            WEBRTC_LINUX
        )
    elseif (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
        target_compile_definitions(libwebrtcbuild
        INTERFACE
            WEBRTC_FREEBSD
        )
    endif()
endif()
```

Reply:
Agreed.

P.S. I can't test that (CMAKE_SYSTEM_NAME STREQUAL "Linux") is an absolute substitute to (LINUX), though.